change/@ni-jasmine-parameterized-5d135b15-d9c6-410c-8cba-5784b0eda870.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add keywords to package metadata",
```

Comment:
Looks like that typo is in the doc comment too: https://github.com/ni/nimble/blob/45c7b88f65076ec1e2b8bb3cd232f37420272ca8/packages/jasmine-parameterized/src/parameterized.ts#L88